src/main/java/com/example/flight/LoadDatabase.java

```diff
@@ -14,11 +16,20 @@
     private static final Logger log = LoggerFactory.getLogger(LoadDatabase.class);
 
     @Bean
-    CommandLineRunner initDatabase(AirportRepository repository) {
+    CommandLineRunner initDatabase(AirportRepository airportRepository, FlightRepository flightRepository) {
 
         return args -> {
-            log.info("Preloading... " + repository.save(new Airport("111A", "Hululu")));
-            log.info("Preloading... " + repository.save(new Airport("222B", "Ulau")));
+            Airport airportA = new Airport("111A", "Hululu");
+            Airport airportB = new Airport("222B", "Ulau");
+            log.info("Preloading... " + airportRepository.save(airportA));
+            log.info("Preloading... " + airportRepository.save(airportB));
+            Flight flightA = new Flight(true, airportA);
+            Flight flightAA = new Flight(false, airportA);
+            Flight flightB = new Flight(true, airportB);
+            log.info("Preloading... " + flightRepository.save(flightA));
+            log.info("Preloading... " + flightRepository.save(flightAA));
+            log.info("Preloading... " + flightRepository.save(flightB));
+
```

Comment:
Either you give space between lines or remove this empty line

Reply:
done